internal/server/server.go

```diff
@@ -0,0 +1,146 @@
+package server
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
+)
+
+type Logger interface {
+	Request(*http.Request, error)
+}
+
+//this is the files in server which need a client
+type Client interface {
+	ErrorHandlerClient
+	DeputyHubInformation
+}
+
+type Template interface {
+	ExecuteTemplate(io.Writer, string, interface{}) error
+}
+
+func New(logger Logger, client Client, templates map[string]*template.Template, prefix, siriusPublicURL, webDir string) http.Handler {
+	wrap := errorHandler(logger, client, templates["error.gotmpl"], prefix, siriusPublicURL)
+
+	mux := http.NewServeMux()
+	mux.Handle("/",
+		wrap(
+			loggingInfoForDeputyHub(client, templates["deputy-hub.gotmpl"])))
+
+	// mux.Handle("/health-check", healthCheck())
```

Comment:
Is this commented out because it is disabled? You should be able to follow the pattern in Workflow to enable it.

Reply:
I didn't implement it because i didn't think it was part of the scope of this ticket - but it's a simple addition so i'll do that now